DataConnectors/CrowdStrike Falcon Endpoint Protection/Connector_Syslog_CrowdStrikeFalconEndpointProtection.json

```diff
@@ -0,0 +1,94 @@
+{
+    "id": "CrowdStrikeFalconEndpointProtection",
+    "title": "CrowdStrike Falcon Endpoint Protection",
+    "publisher": "CrowdStrike",
+    "descriptionMarkdown": "The [CrowdStrike Falcon Endpoint Protection](https://www.crowdstrike.com/endpoint-security-products/) connector allows you to easily connect your CrowdStrike Falcon Event Stream with Azure Sentinel, to create custom dashboards, alerts, and improve investigation. This gives you more insight into your organization's endpoints and improves your security operation capabilities.",
+    "additionalRequirementBanner":"These queries are dependent on a parser based on a Kusto Function to work as expected. Follow the steps to use this Kusto functions alias **CrowdStrikeFalconEventStream** in queries and workbooks. [Follow these steps to get this Kusto functions.](https://aka.ms/sentinel-crowdstrikefalcon-parser)",
+    "graphQueries": [{
+        "metricName": "Total data received",
+        "legend": "CrowdStrikeFalconEventStream",
+        "baseQuery": "CommonSecurityLog​ \n| where DeviceVendor == \"CrowdStrike\" and DeviceProduct == \"FalconHost\""
+    }],
+    "sampleQueries": [{
+        "description": "Top 10 Hosts with Detections",
+        "query": "CrowdStrikeFalconEventStream \n | where EventType == \"DetectionSummaryEvent\" \n| summarize count() by DstHostName /n | top 10 by count_"
+    }, {
+        "description": "Top 10 Users with Detections",
+        "query": "CrowdStrikeFalconEventStream \n | where EventType == \"DetectionSummaryEvent\" \n| summarize count() by DstUserName /n | top 10 by count_"
+    }],
+    "connectivityCriterias": [{
+        "type": "IsConnectedQuery",
+        "value": "CommonSecurityLog​ \n| where DeviceVendor == \"CrowdStrike\" and DeviceProduct == \"FalconHost\"\n | summarize LastLogReceived = max(TimeGenerated)\n | project IsConnected = LastLogReceived > ago(30d)"
+    }],
+    "dataTypes": [{
+        "name": "CommonSecurityLog (CrowdStrikeFalconEventStream)",
+        "lastDataReceivedQuery": "CommonSecurityLog​ \n| where DeviceVendor == \"CrowdStrike\" and DeviceProduct == \"FalconHost\"\n | summarize Time = max(TimeGenerated)\n | where isnotempty(Time)"
+    }],
+    "availability": {
+        "status": 1
```

Comment:
Can you please add "isPreview": true, please refer the template here https://github.com/Azure/Azure-Sentinel/blob/master/DataConnectors/Templates/Connector_CEF_template.json

Reply:
added